set.mm

```diff
@@ -175578,6 +175586,48 @@ equal to the half of the predecessor of the odd number (which is an even
       EFGZWFWKUDVTWBYGWAWCABUIVQCWEUCVRVS $.
   $}
 
+  $( 0 is even.  (Contributed by AV, 11-Feb-2020.)  (Revised by AV,
```

Comment:
for consistency with the following comments:
>  $( 2 divides 0.  That means 0 is even.

Reply:
Alright these suggestions should be applied now